src/main/riddl/Order/Order.riddl

```diff
@@ -4,16 +4,16 @@ context OrderContext is {
   type LineItem is {
     product: SKU,
     quantity: Number,
-    unitPrice: Number,
```

Comment:
I think that maybe you have this a bit backward. unitPrice is the price of quantity 1. lineTotal = quantity * unitPrice. If we were to delete one of the two attributes I would delete lineTotal because it is a calculated attribute.

Reply:
Ahh interesting. Currently in the code base, the order has an orderTotal attribute. This is a calculated attribute in which orderTotal = summation of lineTotal * quantity per line item. Is this wrong?

should it be orderTotal = summation of lineTotal per line item, in which each lineTotal = quantity * unitPrice?

Reply:
Yes it should ^ :p

Reply:
@miguelTaningcoYoppworks - is this fixed yet?

Reply:
Left a direct message to Reid asking how calculated fields must be documented on riddl or how fields only present internally must be documented.

Reply:
For now we are just going to add some documentation to say these are calculated